src/App.tsx

```diff
@@ -34,7 +34,7 @@ try {
   settings = {};
 }
 
-log.setLevel("warn");
+log.setLevel("debug");
```

Comment:
We don't want the default logging level to be `default`.

Reply:
My mistake, forgot to change that back!